repos/fdbt-site/src/pages/api/csvZoneUpload.ts

```diff
@@ -239,42 +239,65 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
                 setFareZoneAttributeAndRedirect(req, res, errors);
                 return;
             }
+            const ticket = getSessionAttribute(req, MATCHING_JSON_ATTRIBUTE);
+
+            const matchingJsonMetaData = getSessionAttribute(req, MATCHING_JSON_META_DATA_ATTRIBUTE);
+            // check to see if we are in edit mode
+            if (ticket && matchingJsonMetaData) {
+                // overriding the existing object
+                const updatedTicket = {
+                    ...ticket,
+                    zoneName: userFareZones[0].FareZoneName,
+                    stops,
+                    exemptedServices: selectedServices,
```

Comment:
dont we only want to add the exemptedServices key if selectedServices.length > 0?